pkg/setting/setting.go

```diff
@@ -179,6 +179,10 @@ var (
 	AlertingErrorOrTimeout     string
 	AlertingNoDataOrNullValues string
 
+	AlertingEvaluationTimeoutSeconds   int
```

Comment:
Suggest using `time.Duration` as type instead of `int` for this and the AlertingNotificationTimeoutSeconds settings. This way you don't have to convert it to `time.Duration` in multiple places. Thinking about it I think these variables don't need to end with seconds - only important that the configuration settings ends with seconds. Make sense?

Reply:
That's really better, I agree